diff --git a/index.js b/index.js
index c425f1ee9d0944b1e2274ebb78528febf563d17e..2ee7a5c40090311ac8971a62f30d838d51ae5181 100644
--- a/index.js
+++ b/index.js
@@ -12,10 +12,9 @@
  * @private
  */
 
-var deprecate = require('depd')('http-errors')
-var setPrototypeOf = require('setprototypeof')
+var setPrototypeOf = Object.setPrototypeOf
 var statuses = require('statuses')
-var inherits = require('inherits')
+var inherits = require('util').inherits
 var toIdentifier = require('toidentifier')
 
 /**
@@ -69,10 +68,6 @@ function createError () {
     }
   }
 
-  if (typeof status === 'number' && (status < 400 || status >= 600)) {
-    deprecate('non-error status code; use only 4xx or 5xx status codes')
-  }
-
   if (typeof status !== 'number' ||
     (!statuses.message[status] && (status < 400 || status >= 600))) {
     status = 500
