diff --git a/encodings/dbcs-codec.js b/encodings/dbcs-codec.js
index fa839170367b271072dc097d29b2c05f085e7681..03616ca08753e29ed31ceb327da25abe58de4bfe 100644
--- a/encodings/dbcs-codec.js
+++ b/encodings/dbcs-codec.js
@@ -1,5 +1,4 @@
 "use strict";
-var Buffer = require("safer-buffer").Buffer;
 
 // Multibyte codec. In this scheme, a character is represented by 1 or more bytes.
 // Our codec supports UTF-16 surrogates, extensions for GB18030 and unicode sequences.
diff --git a/encodings/internal.js b/encodings/internal.js
index dc1074f04f11a31c0e962846f5d162eab9556d38..61f574b8121e4d8b417f3058597b1ff7c62c88f0 100644
--- a/encodings/internal.js
+++ b/encodings/internal.js
@@ -1,5 +1,4 @@
 "use strict";
-var Buffer = require("safer-buffer").Buffer;
 
 // Export Node.js internal encodings.
 
diff --git a/encodings/sbcs-codec.js b/encodings/sbcs-codec.js
index abac5ffaac97da29fa5c5d8aedf5b47763fc7c58..56d6d49746b332213177e31b15c4f0d920d70124 100644
--- a/encodings/sbcs-codec.js
+++ b/encodings/sbcs-codec.js
@@ -1,5 +1,4 @@
 "use strict";
-var Buffer = require("safer-buffer").Buffer;
 
 // Single-byte codec. Needs a 'chars' string parameter that contains 256 or 128 chars that
 // correspond to encoded bytes (if 128 - then lower half is ASCII). 
diff --git a/encodings/utf16.js b/encodings/utf16.js
index 97d066925bbd5dfaa7213e0433570a113c461f3e..df1ba233fc96924709ce7ceadead8477409cfb4f 100644
--- a/encodings/utf16.js
+++ b/encodings/utf16.js
@@ -1,5 +1,4 @@
 "use strict";
-var Buffer = require("safer-buffer").Buffer;
 
 // Note: UTF16-LE (or UCS2) codec is Node.js native. See encodings/internal.js
 
diff --git a/encodings/utf32.js b/encodings/utf32.js
index 2fa900a12eb3562e38fc9442dd3f57ea919b3c74..c80e72d32eacdde6d1fdfcece7fa4aa5f60ea742 100644
--- a/encodings/utf32.js
+++ b/encodings/utf32.js
@@ -1,7 +1,5 @@
 'use strict';
 
-var Buffer = require('safer-buffer').Buffer;
-
 // == UTF32-LE/BE codec. ==========================================================
 
 exports._utf32 = Utf32Codec;
diff --git a/encodings/utf7.js b/encodings/utf7.js
index eacae34d5f80d0b406ad63104406ddd5f3232f4a..ad0bf4fc39c4fab1a92724c8ab3fa283e1d42c13 100644
--- a/encodings/utf7.js
+++ b/encodings/utf7.js
@@ -1,5 +1,4 @@
 "use strict";
-var Buffer = require("safer-buffer").Buffer;
 
 // UTF-7 codec, according to https://tools.ietf.org/html/rfc2152
 // See also below a UTF-7-IMAP codec, according to http://tools.ietf.org/html/rfc3501#section-5.1.3
diff --git a/lib/index.js b/lib/index.js
index 657701c38d243b8af1cd3d4a67056e095a0ede5e..f224ea462925af6ae4fb753e193b76eb83ed3e5d 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -1,7 +1,5 @@
 "use strict";
 
-var Buffer = require("safer-buffer").Buffer;
-
 var bomHandling = require("./bom-handling"),
     iconv = module.exports;
 
diff --git a/lib/streams.js b/lib/streams.js
index a1506482f580162d5b3a07b1ef82fcca22b40e5a..bd00f0674054c0d9d5ddb8cb5852686da71fe5f2 100644
--- a/lib/streams.js
+++ b/lib/streams.js
@@ -1,7 +1,5 @@
 "use strict";
 
-var Buffer = require("safer-buffer").Buffer;
-
 // NOTE: Due to 'stream' module being pretty large (~100Kb, significant in browser environments), 
 // we opt to dependency-inject it instead of creating a hard dependency.
 module.exports = function(stream_module) {
