diff --git a/picocolors.js b/picocolors.js
index fdb630451d89b134b069355bb97556e39b0b171a..389dee8a5d5c91c66a480f8230242b651deddce3 100644
--- a/picocolors.js
+++ b/picocolors.js
@@ -52,6 +52,12 @@ let createColors = (enabled = isColorSupported) => ({
 	bgMagenta: enabled ? formatter("\x1b[45m", "\x1b[49m") : String,
 	bgCyan: enabled ? formatter("\x1b[46m", "\x1b[49m") : String,
 	bgWhite: enabled ? formatter("\x1b[47m", "\x1b[49m") : String,
+	redBright: enabled ? formatter("\x1b[91m", "\x1b[39m") : String,
+	greenBright: enabled ? formatter("\x1b[92m", "\x1b[39m") : String,
+	yellowBright: enabled ? formatter("\x1b[93m", "\x1b[39m") : String,
+	blueBright: enabled ? formatter("\x1b[94m", "\x1b[39m") : String,
+	magentaBright: enabled ? formatter("\x1b[95m", "\x1b[39m") : String,
+	cyanBright: enabled ? formatter("\x1b[96m", "\x1b[39m") : String,
 })

 module.exports = createColors()
diff --git a/types.ts b/types.ts
index b4bacee4909e7f562fb13f89720c8ae57c4922fc..7b8327138390bfa3c397690a62a2c269cdbac06e 100644
--- a/types.ts
+++ b/types.ts
@@ -27,4 +27,10 @@ export interface Colors {
 	bgMagenta: Formatter
 	bgCyan: Formatter
 	bgWhite: Formatter
+	redBright: Formatter
+	greenBright: Formatter
+	yellowBright: Formatter
+	blueBright: Formatter
+	magentaBright: Formatter
+	cyanBright: Formatter
 }
